### PR TITLE
Feature/custom providers

### PR DESCRIPTION
**Pull request by fimak**
Custom providers + bugs + disabled sms auth

### CODE REVIEW

File: app/screens/CustomProviders/CustomProvidersScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  'twitter',
+  'discord',
+  'line',
+  'reddit',
+  'apple',
+  'github',
+  'twitch',
+  'linkedin',
+  'weibo',
+];
```
Comment (Royalone94) [inline app/screens/CustomProviders/CustomProvidersScreen.tsx:27]: can we put these variables into any constants file?

File: app/screens/CustomProviders/CustomProvidersScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        //   if (response.userInfo.verifierId[0] === '+') {
+        //     // check mobile number
+        //     const phoneNumber = response.userInfo.verifierId.slice(1).replace('-', '');
+        //     const eSignatureUser = await getUserByWallet(login);
+        //   } else {
+        //     // check email
+        //   }
+        // }
+        //
+      })
```
[the remark this answers is not in the document]
Comment (Royalone94) [reply app/screens/CustomProviders/CustomProvidersScreen.tsx:81]: It seems it is just for sms feature, and disabled.  let me skip that for now.

[the remark this answers is not in the document]
Comment (fimak) [reply app/screens/CustomProviders/CustomProvidersScreen.tsx:81]: We need it in the next release.
I commented some code which is related to sms auth.

File: app/screens/CustomProviders/CustomProvidersScreen.tsx (hunk 1 above)
Comment (fimak) [reply app/screens/CustomProviders/CustomProvidersScreen.tsx:27]: sure

Comment (fimak) [reply app/screens/CustomProviders/CustomProvidersScreen.tsx:27]: Fixed

**Pull request merged**